com.unity.render-pipelines.universal/Shaders/SpatialMappingWireframe.shader

```diff
@@ -0,0 +1,156 @@
+Shader "Universal Render Pipeline/SpatialMappingWireframe"
+{
+    Properties
+    {
+        _WireThickness ("Wire Thickness", RANGE(0, 800)) = 100
+    }
+    SubShader
+    {
+        Tags {"RenderType" = "Opaque" "IgnoreProjector" = "False" "RenderPipeline" = "UniversalPipeline" "ShaderModel"="2.0"}
+        LOD 100
+
+        Pass
+        {
+            Name "Spatial Mapping Occlusion"
+
+            HLSLPROGRAM
+            #pragma require geometry
+            #pragma exclude_renderers gles gles3 glcore
+            #pragma target 4.5
```

Comment:
these two lines seem too restrictive? do these line up with the original shader they are based off in builtin?